src/components/Wizards/CreateVmWizard/NetworksTab.js

```diff
@@ -196,10 +198,13 @@ export class NetworksTab extends React.Component {
         {
           id: 'network-edit',
           type: 'dropdown',
-          choices: this.props.networkConfigs.map(networkConfig => networkConfig.metadata.name).filter(networkConfig => {
-            const result = this.state.rows.find(r => r.network === networkConfig);
-            return !result;
-          }),
+          choices: this.props.networkConfigs
+            .map(networkConfig => networkConfig.metadata.name)
+            .concat(POD_NETWORK)
+            .filter(networkConfig => {
+              const result = this.state.rows.find(r => r.network === networkConfig);
```

Comment:
Can this be one-liner?

Reply:
@mareklibra changed to one-liner and rebased on master